mta-sts.md

```diff
@@ -153,7 +153,7 @@ semicolon-separated key/value pairs containing the following fields:
 
 An example TXT record is as below:
 
-`_mta-sts.example.com.  IN TXT "v=STSv1; id=20160831085700Z;"`
+_mta-sts.example.com.  IN TXT "v=STSv1; id=20160831085700Z;"
```

Comment:
I do not really understand what this change does. I thought the outer ticks were stripped in the rendering and just made this a "code" block?

Reply:
It seems to have been making it double quotes for some reason.